projects/plugins/crm/src/automation/data-transformers/class-data-transformer-invoice-to-contact.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * CRM Invoice to CRM Contact Transformer class.
+ *
+ * @package Automattic\Jetpack\CRM
+ */
+
+namespace Automattic\Jetpack\CRM\Automation\Data_Transformers;
+
+use Automattic\Jetpack\CRM\Automation\Data_Transformer_Exception;
+use Automattic\Jetpack\CRM\Automation\Data_Types\Data_Type_Base;
+use Automattic\Jetpack\CRM\Automation\Data_Types\Data_Type_Contact;
+use Automattic\Jetpack\CRM\Automation\Data_Types\Data_Type_Invoice;
+
+/**
+ * CRM Invoice to CRM Contact Transformer class.
+ *
+ * @since $$next-version$$
+ *
+ * @package Automattic\Jetpack\CRM\Automation
```

Comment:
Should we use @package in classes too? Or only in the file header as per WP Coding standard expects.

Reply:
There are more classes in this PR with the @package tag added to their PHPDoc.